### DISCUSSION TITLE
Is there a reason the text doesn't go far to the right?

### DISCUSSION THREAD

**Discussion by peyre** [raceintospace/raceintospace#579] [Q&A]
I've wondered why the text in the newscast wraps so far to the left.  Is there a reason we can't move that over so you'd have more single-line items?

![Let text go further to the Right](https://user-images.githubusercontent.com/3118014/132266204-3e8d885b-b37b-4ce1-b692-5307a001ae8b.png)

**Comment by peyre** [comment]
Oh, I wonder if it's to make sure long names (like Rozhdestvensky) don't cause the text to bleed over into the scroll bar.  Still, there should be room to fit 
ROZHDESTVENSKY WAS REMOVED FROM CURRENT FLT CREW
without it bleeding over.  Maybe we could extend the righthand margin?

**Reply by rnyoakum** [reply]
It appears to be an attempt to make sure the text doesn't bleed into the scroll bar. The right-hand cutoff appears to be 35 characters. The next time a space (' ') is encountered after 35 characters or more have been printed, it wraps to a new line. The code is in src/game/news.cpp, in the function `DispNews`.

**Reply by peyre** [reply]
Hey, thanks.  I think I'll tinker with it this morning.

**Comment by peyre** [comment]
So, I set it to 40 characters instead, and everything looked really good!
![News](https://user-images.githubusercontent.com/3118014/145727689-51ec54eb-d21f-443b-a8f8-657b58fb8ac5.png)
![News2](https://user-images.githubusercontent.com/3118014/145727665-e7b7f6de-77d5-42f3-8c6c-3b212e0ff2f8.png)

But then I eventually ran into this, so I see why the cutoff has to be so low (unless we can fix it somehow).
![SeeWhatYouMean](https://user-images.githubusercontent.com/3118014/145727680-2424ed78-c39c-4f79-b9c9-b5cb345bacba.png)

**Reply by rnyoakum** [reply]
The game could look a word ahead and determine if it's going to overflow the line past some arbitrary length, and if so, wrap to a new line. That's what I would look at doing next.

**Reply by peyre** [reply]
Oh yeah, maybe it could change the equation to if (k + ((length of Data->P[plr].Pool[j].Name) * 8) > 44) or something like that.
